extensions/amp-app-banner/1.0/base-element.js

```diff
@@ -0,0 +1,25 @@
+import {PreactBaseElement} from '#preact/base-element';
+
+import {BentoAppBanner} from './component/component';
+import {CSS as COMPONENT_CSS} from './component/component.jss';
+
+export class BaseElement extends PreactBaseElement {}
+
+/** @override */
+BaseElement['Component'] = BentoAppBanner;
+
+/** @override */
+BaseElement['props'] = {
+  'children': {passthrough: true},
+  'dismissButtonAriaLabel': {attr: 'dismiss-button-aria-label'},
+  'id': {attr: 'id'},
```

Comment:
is the `id` prop used/necessary?

Reply:
Yes, it's required (it's used to generate the localStorage key)